utils.py

```diff
@@ -318,12 +317,9 @@ def __init__(self, check_id, *args):
         self.id = check_id # pylint: disable=invalid-name
 
 
-def setup_logger():
-    """
-    Initialiser le logger
-
-    :return: None
-    """
+# pylint: disable=pointless-string-statement
```

Comment:
On utilise pas des strings pour commenter du code 😭

Reply:
Pas moi qui l'ai fait et si tous les IDE le proposent

Reply:
C'est une bad practice car Python vas tenter de l'exécuter comme une instruction, ça be sert à rien, mais ça peut légèrement ralentir l'exécution d'un programme (pour Gipsy ça aura pas d'impact, mais quand tu fais du circuitPython sur un Arduino tu fais gaffe à tout)

Reply:
If you Walt, de toute manière on a gît donc autant l'utiliser